client/modules/core/main.js

```diff
@@ -375,39 +396,69 @@ export default {
     });
   },
 
-  // primaryShopId is the first created shop. In a marketplace setting it's
-  // the shop that controls the marketplace and can see all other shops.
+  /**
+   * primaryShopId
+   * @summary getter method for the first created shop. In a marketplace setting
+   * it's the shop that controls the marketplace and can see all other shops.
+   * @param {Number} the document id for the Primary Shop
+   */
   get primaryShopId() {
     return this._primaryShopId.get();
   },
 
+  /**
+   * primaryShopId
+   * @summary setter method for the primary shop.
+   * (should probably only be used at startup)
+   */
   set primaryShopId(shopId) {
     this._primaryShopId.set(shopId);
   },
 
-  getPrimaryShopId() {
-    return this.primaryShopId;
```

Comment:
_(this is just some strange GH highlighting... the code is not removed, it is below)_